modules/core/04-channel/keeper/packet.go

```diff
@@ -151,7 +151,15 @@ func (k Keeper) SendPacket(
 		),
 	})
 
-	k.Logger(ctx).Info("packet sent", "packet", fmt.Sprintf("%v", packet))
+	k.Logger(ctx).Info(
+		"packet sent",
+		"sequence", packet.GetSequence(),
+		"src_port", packet.GetSourcePort(),
+		"src_channel", packet.GetSourceChannel(),
+		"dst_port", packet.GetDestPort(),
+		"dst_channel", packet.GetDestChannel(),
+		"data", fmt.Sprintf("%X", packet.GetData()),
```

Comment:
we can omit this if necessary.

Reply:
Is logging primarily used for debugging?

Omitting this seems like the safer option to me (we can add it back if we find a good reason to). Logging the packet data seems like a nice to have to me since you'd still need to decode it into the correct packet data type to understand its contents. Relayers will have event logs which should include the packet data

Reply:
Debugging and basic telemetry into the operation of the process. I'll remove `data`.